RiotSwiftUI/Modules/Room/VoiceBroadcastRecorder/Coordinator/VoiceBroadcastRecorderProvider.swift

```diff
@@ -83,4 +95,17 @@ import Foundation
 
         return coordinatorsForEventIdentifiers[currentEventIdentifier]
     }
+    
+    private func handleEvent(event: MXEvent, direction: MXTimelineDirection, customObject: Any?) {
```

Comment:
perhaps we can name this method handleRedactedEvent as it's only process those events.